### ISSUE TITLE
licensing problems

### ISSUE THREAD

**Issue by aissat** [unity8org/unity8org#6]
I think it is important to make sure that the code is licensed, to keep away from future licensing problems

**Comment by jsalatas** [comment]
Yes! Actually the code is distributed under GPL. I would appreciate howvere if someone could take a look at other issues.

**Comment by jsalatas** [comment]
Can we please continue the discussion at the mailing list? :)

https://unity8.org/cgi-bin/mailman/listinfo/dev

**Comment by AaronBottegal** [comment]
Which GPU? GPL v1, v2, or vcancer? You have to be specific.

**Comment by jsalatas** [comment]
@AaronBottegal the one already included in the forked repo (see COPYING and COPYING.LGPL)

**Comment by AaronBottegal** [comment]
Okay, I know it wasn't just GPL, just asking to make sure as I didn't hit the repo yet specifically.

**Comment by aissat** [comment]
I think it is best to clean the code  from every possible line causing future licensing problems

**Comment by jsalatas** [comment]
I was hopping doing it later today.... :)

**Comment by aissat** [comment]
@jsalatas great ☺️

**Comment by KAMiKAZOW** [comment]
Existing code is GPLv3-only. Other projects in similar situation (most notably LibreOffice that forked LGPLv3-only code) adopted a policy that new code should include the "or any later option" clause for flexibility into the future.

**Comment by jsalatas** [comment]
I'm leaving this issue open, as I'm considering any licensing violations  to be a bug in the software that needs to be addressed

**Comment by bregma** [comment]
At Canonical we're pretty darn strict about licensing in out code -- we have to be, because it's widely-used software in full public view.  What sort of license violations do anticipate?

I would be more concerned about trademarks.  For that, you would do well to talk to Canonical to either get permission or to avoid violation.

If you don't know the difference between copyright, distribution licenses, and trademarks, you might want to get some legal advice. Here be dragons.

**Comment by stavpup** [comment]
GPL v3 or later should be sufficient.

**Comment by jsalatas** [comment]
This is not the point. The code is already in GPLv3 and it is not sufficient. We need to find someone who speaks legaleze though :)

**Comment by star-buck** [comment]
I thought one could not impose any other additional licensing restrictions on GPLv3'd code?

**Comment by jsalatas** [comment]
It has to do about trademarks, not actual restrictions to the code. See for example the fork of X11 (which was a registered trademark) to X.org.

**Comment by step21** [comment]
As someone said above, please do not confuse licensing and trademarks, or other issues. As was pointed out above, there should not be any issues with the code, as long as you adhere to GPL v3.
Regarding trademarks, you should talk to canonical, if they are no longer developing Unity they might just allow you to use it. But better be safe and ask.

**Comment by jsalatas** [comment]
I talked today with the CEO of "Law and Tech" (http://lawandtech.eu/en/) who are attorneys specializing in technology related issues. He informed me that we obviously need to strip away any trademarks (including logos) from the code. Regarding the look and feel part (I'm quoting here) "Minor changes are sufficient".

**Comment by reinhart1010** [comment]
>  we obviously need to strip away any trademarks (including logos) from the code.
Actually, I still don't know whether we can use the "Ubuntu" font anymore. In fact, the font also contain the Ubuntu Logo. Here's the results of comparing "Ubuntu" with "[Aileron](https://fontlibrary.org/en/font/aileron)" font, which has slightly narrower characters than Ubuntu, without containing Ubuntu logos and has a [CC0](https://creativecommons.org/publicdomain/zero/1.0/) license instead.
![ubuntu aileron](https://cloud.githubusercontent.com/assets/17312341/25048343/40c4db72-2166-11e7-9299-f60db2c944e3.png)

**Comment by jsalatas** [comment]
ubuntu font is fine. It's a free font (I guess) :\

**Comment by WernerLDev** [comment]
From the ubuntu font website: 
'Both the final font Truetype/OpenType files and the design files used to produce the font family are distributed under an open licence and you are expressly encouraged to experiment, modify, share and improve. "

I don't think it's a problem if we use the ubuntu font. And personally I quite like the font. :-)

**Comment by KAMiKAZOW** [comment]
You can use the font (many Linux distributions already package it, Ubuntu obviously does) but you can't use the Ubuntu logo in the dock, documentation, etc.

For inspiration how to properly describe your project without violating trademarks, have a look at https://www.centos.org/about/ which is in a similar situation with regards to Red Hat's trademarks.
